llvm/lib/Target/RISCV/RISCVSystemOperands.td

```diff
@@ -351,7 +364,10 @@ def SysRegVLENB: SysReg<"vlenb", 0xC22>;
 //===----------------------------------------------------------------------===//
 // Shadow Stack CSR
 //===----------------------------------------------------------------------===//
-def : SysReg<"ssp", 0x011>;
+
+let FeaturesRequired = [{ {RISCV::FeatureStdExtZicfiss} }] in {
+  def : SysReg<"ssp", 0x011>;
```

Comment:
The extension is experimental in LLVM due to incomplete implementation of CodeGen support for Zicfiss. The extension itself is ratified so the CSR address is frozen. Should we consider assembly support for Zicfiss as non-experimental?

Reply:
My initial instincts are to say that when it comes to CSRs experimental should be reserved solely for ones that don't have ratified encodings

Reply:
I see your point, while we don't have any CSR that is unratified now, so I simply think it's a way to show how things work in the code. Else this patch turns into a error message update and we should consider document or comment it somewhere for developers to notice